### PR TITLE
update spreedme

### PR DESCRIPTION
**Pull request by enoch85**
Merge before new release to not mess up current updates

### CODE REVIEW

**Comment by enoch85** [comment]
@oparoz @ezraholm50 @Pe46dro I need a review on this. Thanks!

**Comment by oparoz** [comment]
Cool about the config. Do you include spreedme.conf in the Nextcloud config?

**Comment by enoch85** [comment]
@oparoz Yes I do. :) Did you test it?

**Comment by enoch85** [comment]
Tried to install it on my system and got this:

```
{"reqId":"PkQkd95oxOVhKsOhokBE","remoteAddr":"192.168.4.201","app":"index","message":"Exception: {\"Exception\":\"ParseError\",\"Message\":\"syntax error, unexpected 'cdee01aad0b49b3f998fa23aedda90' (T_STRING), expecting ',' or ';'\",\"Code\":0,\"Trace\":\"#0 [internal function]: OC\\\\Autoloader->load('OCA\\\\\\\\SpreedME\\\\\\\\Co...')\\n#1 [internal function]: spl_autoload_call('OCA\\\\\\\\SpreedME\\\\\\\\Co...')\\n#2 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/spreedme\\\/helper\\\/helper.php(127): class_exists('\\\\\\\\OCA\\\\\\\\SpreedME\\\\\\\\C...', true)\\n#3 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/spreedme\\\/helper\\\/helper.php(121): OCA\\\\SpreedME\\\\Helper\\\\Helper::doesPhpConfigExist()\\n#4 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/spreedme\\\/controller\\\/pagecontroller.php(29): OCA\\\\SpreedME\\\\Helper\\\\Helper::notifyIfAppNotSetUp()\\n#5 [internal function]: OCA\\\\SpreedME\\\\Controller\\\\PageController->__construct('spreedme', Object(OC\\\\AppFramework\\\\Http\\\\Request), 'daniel@en0ch.se')\\n#6 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/AppFramework\\\/Utility\\\/SimpleContainer.php(67): ReflectionClass->newInstanceArgs(Array)\\n#7 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/AppFramework\\\/Utility\\\/SimpleContainer.php(84): OC\\\\AppFramework\\\\Utility\\\\SimpleContainer->buildClass(Object(ReflectionClass))\\n#8 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/AppFramework\\\/Utility\\\/SimpleContainer.php(105): OC\\\\AppFramework\\\\Utility\\\\SimpleContainer->resolve('OCA\\\\\\\\SpreedME\\\\\\\\Co...')\\n#9 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/AppFramework\\\/App.php(98): OC\\\\AppFramework\\\\Utility\\\\SimpleContainer->query('OCA\\\\\\\\SpreedME\\\\\\\\Co...')\\n#10 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/AppFramework\\\/Routing\\\/RouteActionHandler.php(47): OC\\\\AppFramework\\\\App::main('OCA\\\\\\\\SpreedME\\\\\\\\Co...', 'index', Object(OC\\\\AppFramework\\\\DependencyInjection\\\\DIContainer), Array)\\n#11 [internal function]: OC\\\\AppFramework\\\\Routing\\\\RouteActionHandler->__invoke(Array)\\n#12 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/Route\\\/Router.php(293): call_user_func(Object(OC\\\\AppFramework\\\\Routing\\\\RouteActionHandler), Array)\\n#13 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/base.php(983): OC\\\\Route\\\\Router->match('\\\/apps\\\/spreedme\\\/')\\n#14 \\\/var\\\/www\\\/nextcloud\\\/index.php(48): OC::handleRequest()\\n#15 {main}\",\"File\":\"\\\/var\\\/www\\\/nextcloud\\\/apps\\\/spreedme\\\/config\\\/config.php\",\"Line\":28}","level":3,"time":"2016-10-26T14:09:49+02:00","method":"GET","url":"\/index.php\/apps\/spreedme\/","user":"daniel"}
``` 

Will keep investigating.

**Comment by enoch85** [comment]
Ok, so the error is fixed by adding " " in the right place, but now I get a blank screen instead.

**Comment by enoch85** [comment]
Now I get this in Apache2 error log:

```
[Wed Oct 26 14:29:20.900672 2016] [proxy:error] [pid 7555] AH00961: HTTPS: failed to enable ssl support for 127.0.0.1:8080 (127.0.0.1)

```

@oparoz Any ideas?

**Comment by enoch85** [comment]
Now it says that `http://192.168.4.15/webrtc/extra/static/config/OwnCloudConfig.js` not is found (404) but it exists.

**Comment by enoch85** [comment]
Ok, `http://192.168.4.15/webrtc/extra/static/config/OwnCloudConfig.js.in` is found.

I guess it has something to do wit the snap? @oparoz

File: nextcloud_update.sh [hunk 1]
```diff
@@ -106,6 +106,8 @@ fi
 chown -R root:root $BACKUP
 
 # Enable Apps
+snap refresh spreedme
+sudo -u www-data php $NCPATH/occ app:enable spreedme
```
Comment (Pe46dro) [inline nextcloud_update.sh]: The script should enable app only on if spreedme is installed

Comment (enoch85) [inline nextcloud_update.sh]: Exactly my thought. I was actually about to add it soon.

**Comment by enoch85** [comment]
@Pe46dro Does the current spreedme.sh work for you?

**Comment by enoch85** [comment]
I merged the nextcloud_update fix already in a separete commit. Please test if spreedme.sh works for you so that we can get this in. @Pe46dro @ezraholm50 @oparoz

**Comment by enoch85** [comment]
@ezraholm50 What do you mean? Test the current spreedme.sh script as it is right now. Don't mind about the update script, that's already merged in a separate commit. :)

**Comment by enoch85** [comment]
This works locally on the VM, just tested. Must be something with my own settings. Let's merge this! \o/

**Pull request merged**